package.json

```
@@ -48,7 +49,7 @@
     "url": "https://github.com/vuejs/vitepress/issues"
   },
   "engines": {
-    "node": ">=14.6.0"
+    "node": "^14.18.0 || >=16.0.0"
```

This seems like it could be simplified:
```suggestion
    "node": ">=14.18.0"
```

Unless you were specifically trying to disallow v15. But in that case I'd expect v17 to be disallowed as well

---

This is what Vite has (https://github.com/vitejs/vite/blob/main/packages/vite/package.json#L35). I don't think even if we allow it, it can run Vite.

---

Maybe we don't need to list anything then and can just rely on Vite's?